config/pass.js

```diff
@@ -24,6 +24,46 @@ module.exports = function(passport, LocalStrategy) {
     }
   ));
 
```

Comment:
Repcoin has two apps registered with facebook now. One is prod and one is dev (both hosted with my facebook account).

An app can only have one URL associated with it. This check allows us to use the facebook login code whether we are on localhost (not production) or heroku (production).